server/libs/logger/logger.ts

```diff
@@ -1,2 +1,30 @@
-// eslint-disable-next-line no-console
-export const logger = () => console.log('TODO: IMPLEMENT LOGGER')
+import { pino, stdTimeFunctions } from 'pino'
+import type { Logger, LoggerOptions } from 'pino'
+
+import { pinoPrettyTransport } from './transports'
+
+const isProd = process.env.NODE_ENV === 'production'
```

Comment:
we keep making this check throughout. would be useful/beneficial to just make it once in particular file and import as needed?

Reply:
I thought the same but how both loggers can have different configuration I decided to separate it. I will upload a change merging both configurations and we can decide 👍

Reply:
Now I read correctly your comment. Yeah, totally right, I will check if Mathigon provides us with that functionality if not I can create some kind of utility.